src/Raven.Server/Documents/Replication/ReplicationBatchDocumentItem.cs

```diff
@@ -22,5 +22,48 @@ public class ReplicationBatchDocumentItem
         public DocumentFlags Flags;
         public short TransactionMarker;
         public long LastModifiedTicks;
+
```

Comment:
Don't use implicit operators, they are a good way to get into head scratching contests.
Use explicit `From` method overloads.